discovery_engine/lib/src/domain/models/document.dart

```diff
@@ -31,21 +31,24 @@ class Document {
   @HiveField(0)
   final DocumentId documentId;
   @HiveField(1)
-  final WebResource webResource;
+  final StackId stackId;
```

Comment:
Now this is ok but in the future we will not be able to shuffle the field ids.

Reply:
yes, I'm aware, but I thought it's ok, since we didn't release the app anywhere yet

Reply:
Yep now is ok